src/kibana/plugins/dashboard/styles/main.less

```diff
@@ -72,8 +72,9 @@ dashboard-grid {
 
         .panel-title {
           font-size: inherit;
+          display: inline-block;
```

Comment:
per Spencer:
"inline elements automatically participate in flexbox layout when contained in a flexed parent element, so inline-block isn't needed"